nginx.agent.conf

```diff
@@ -0,0 +1,23 @@
+include common/main.conf;
+
+
+http {
+    upstream dddt {
+        server unix:/run/dcos/3dt.sock;
+    }
+
+    include common/http.conf;
+
+    server {
+        server_name agent.mesos;
+        include common/server.conf;
+
+        listen 61001 default_server;
+
+        location /system/health/v1 {
```

Comment:
The definition for `/system/health/v1` is the same for both `nginx.master.conf` and `nginx.agent.conf` (except for a JWT auth step that gets no-opped due to disabled security anyway.) So this location can be DRY'ed out of the individual conf files and into `common/server.conf`.